pages/argent.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import type { NextPage } from "next";
+import homeStyles from "../styles/Home.module.css";
+import DiscountEndScreen from "../components/discount/discountEndScreen";
+import DiscountOfferScreen from "../components/discount/discountOfferScreen";
+
+// Create a new discount in utils to create a new discount campaign
+import { argentDiscount } from "../utils/discounts/argent";
+import RegisterDiscount from "@/components/discount/registerDiscount";
+import styles from "../styles/discount.module.css";
+import { useAccount, useConnect } from "@starknet-react/core";
+import ConnectButton from "@/components/UI/connectButton";
+
+const Argent: NextPage = () => {
+  const [searchResult, setSearchResult] = useState<SearchResult | undefined>();
+  const [screen, setScreen] = useState<number>(1);
+  const { account } = useAccount();
+  const connector = useConnect();
+
+  useEffect(() => {
+    const currentDate = new Date();
+    const timestamp = currentDate.getTime();
+
+    if (timestamp >= argentDiscount.expiry) {
+      setScreen(0);
+    }
+  }, []);
+
+  const isArgent = ["Argent X", "Argent (mobile)"].includes(
+    connector.connector?.name ?? ""
+  );
```

Comment:
As said in the E2E you need to give feedback saying this campaign is only for Argent when the user has another wallet.